src/components/filters/ActiveFilterList.vue

```diff
@@ -41,9 +41,12 @@
       removeFilter (filterType, selectedFilterId) {
         const filters = utils.removeFilterFromFilterArrayById(this.activeFilters[filterType], selectedFilterId)
         this.$store.commit(UPDATE_FILTER, {name: filterType, filters: filters})
+        const value = filters.length === 0 ? undefined : filters.join(',')
+        this.$router.push({query: {...this.$store.state.route.query, [filterType]: value}})
       },
       resetAllFilters () {
         this.$store.commit(RESET_FILTERS)
+        this.$router.push({query: {}})
```

Comment:
The only reason that the route got changed if you changed the filters was that the filter component was watching the filter. I've switched to explicitly updating the route if when the filters get updated.